compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -282,6 +282,14 @@ impl<'a: 'ast, 'ast> LateResolutionVisitor<'a, '_, 'ast> {
                             "you may want to use a bool value instead",
                             format!("{}", item_typo),
                         ))
+                    // FIXME(vicnenzopalazzo): make the check smarter,
+                    // and maybe expand with chalk
```

Comment:
Can you explain the chalk mention?

Reply:
yeah, I need to spend some more time on it but we discuss it with @estebank on Zulip and I think it is a good experiment to add because the error can be also a type like `prin!t` or whatever. However, this diff solution needs to be checked and experimented on a little bit.

I'm not a fan to have a static check in the code like this, but I think this is a good starting point.

P.S: this is done for a [Contributor::new](https://rust-lang.zulipchat.com/#narrow/stream/328238-RustcContributor.3A.3Anew) session.

Reply:
Sorry, I'm just curious what "chalk" means here -- when I see that word I think of https://github.com/rust-lang/chalk, but that's unrelated.

Reply:
Oh, damt! I think it is levenshtein distance checks, thanks for the good catch!